README.md

```diff
@@ -41,6 +41,15 @@ agent.registerMySQL() # instrument the MySQL client
 ...
 ```
 
+or
+
+- Use the autoinstrumentation CLI
+```
+HT_INSTRUMENTED_MODULES=flask,mysql
+hypertrace-instrument python app.py
```

Comment:
we should document starting up with `hypertrace-instrument flask run` to avoid confusing people accustomed to launching flask apps using the flask cli